samples/ipc/ipc_service/remote/prj_icmsg.conf

```diff
@@ -4,3 +4,9 @@ CONFIG_PRINTK=y
 CONFIG_LOG=y
 
 CONFIG_APP_IPC_SERVICE_SEND_INTERVAL=25
+
+CONFIG_HEAP_MEM_POOL_SIZE=4096
```

Comment:
heap mem poll size could also be decreased for smaller targets 2k maybe would be enough?

Reply:
done